README.md

```diff
@@ -372,22 +373,41 @@ jobs:
 
 Note that if you specify `self-hosted` in your workflow, then this will run your job on _any_ self-hosted runner, regardless of the labels that they have.
 
+## Runner Groups
+
+Runner groups can be used to limit which repositories are able to use the GitHub Runne at an Organisation level.
```

Comment:
typo. -Runne-

Reply:
That was a long Friday. Thanks for spotting that